### ISSUE TITLE
could be possible to implement an "air quality" sensor?

### ISSUE THREAD

**Issue by mrMiimo** [naofireblade/homebridge-weather-plus#169]
Hi everyone,
many services, such as wunderground, provide data for "air quality"
![ScreenShotDec20](https://user-images.githubusercontent.com/49750870/103233151-9e70f380-493c-11eb-87f8-17a36f5d7f21.png)
.
Can this information be used to implement an "air quality" sensor?
I am not a developer but I could do some tests if necessary ...
Thanks!

**Comment by joconor** [comment]
I don't know if this addresses your specific need, but there is a separate Homebridge plugin that will act as an air quality sensor, pulling data from any of several different services: https://www.npmjs.com/package/homebridge-airnow

**Comment by mrMiimo** [comment]
Yes, it could be an option .. But AirNow is limited to US locations only and Aqicn is not available in all locations.
Thanks to you, I searched and found another plugin [https://www.npmjs.com/package/homebridge-airvisual-2], with some other place ...
However I think that a service like wundergrond (with so many weather stations) you can find many places and having everything in one plugin is much more "efficient", isn't it?

**Comment by dacarson** [comment]
AirQuality is a whole HomeKit service, and I believe would appear as a separate service to the weather service. 

I could see adding the AirQuality service as it is part of the data returned Weather Underground. I don't see it on any other service.
